services/core-web/src/components/mine/Permit/MinePermitTable.js

```diff
@@ -80,7 +81,7 @@ const renderDeleteButtonForPermitAmendments = (record) => {
     return;
   }
 
-  const isLinkedToNowApplication = !isEmpty(record.amendmentEdit.amendment.now_application_guid);
+  const isLinkedToNowApplication = !isEmpty(record.operations.amendment.now_application_guid);
 
```

Comment:
you should be able to read this as 'record.now_application_guid` as `amendment` has been spread into the object. 

Can you update and confirm?

Reply:
Updated.